src/dftbp/dftbplus/mainio.F90

```diff
@@ -4014,60 +4023,75 @@ subroutine writeMdOut2(fd, tStress, tPeriodic, tBarostat, isLinResp, eField, tFi
     integer :: ii
     character(lc) :: strTmp
 
-    if (tStress) then
-      if (tBarostat) then
-        write(fd, "(A)") 'Lattice vectors (A)'
+    if (isPeriodic) then
+      if (withBarostat) then
+        write(fd, "(A)") "Lattice vectors (A)"
         do ii = 1, 3
           write(fd, "(3E24.8)") latVec(:,ii) * Bohr__AA
         end do
-        write(fd, format2Ue) 'Volume', cellVol, 'au^3', (Bohr__AA**3) * cellVol, 'A^3'
       end if
-      if (tPeriodic) then
-        write(fd, format2Ue) 'Pressure', cellPressure, 'au', cellPressure * au__pascal, 'Pa'
-        if (pressure /= 0.0_dp) then
-          write(fd, format2U) 'Gibbs free energy', energy%EGibbs, 'H',&
-              & Hartree__eV * energy%EGibbs,'eV'
-          write(fd, format2U) 'Gibbs free energy including KE', energy%EGibbsKin, 'H',&
-              & Hartree__eV * energy%EGibbsKin, 'eV'
-        end if
+      if (printForces) then
```

Comment:
Should this be inside the isPeriodic block, as forces for molecular cases can be requested.

Reply:
No it shouldn't, thanks for the fix.